client/src/featureform/sqlite_metadata.py

```diff
@@ -424,3 +538,29 @@ def search(self, phrase, resource_type=None):
         results_list = search_results.fetchall()
 
         return results_list
+
+    def upsert_tags_properties(self, tablename, *args):
+        query = ""
+        updatable_column = "tag_list" if tablename == "tags" else "property_list"
+        default_value = [] if tablename == "tags" else {}
+        is_update = False
+        name = args[0]
+        variant = args[1]
+
+        cursor = self.__conn.execute(f"SELECT * FROM {tablename} WHERE name = '{name}' AND variant = '{variant}';")
+        self.__conn.commit()
+        existing = cursor.fetchall()
+
+        if len(existing):
+          existing_val = json.loads(existing[0][updatable_column]) if existing[0][updatable_column] is not None else default_value
```

Comment:
Should make const or assign existing[0] to a var instead of using a #. A lot of magic numbers in this function